browser/index.html

```diff
@@ -16,6 +16,7 @@
 <script src="https://cdn.rawgit.com/CamHenlin/TouchPolyfill/master/touchpolyfill.min.js#sabaki-v0.11.0"></script>
 <script src="../node_modules/sigma/build/sigma.min.js"></script>
 <script src="../node_modules/sigma/build/plugins/sigma.renderers.customShapes.min.js"></script>
+<script src="../node_modules/clipboard/dist/clipboard.min.js"></script>
```

Comment:
Instead of including clipboard.js with a `<script>` tag, I'd rather have it required as a node.js module in `index.js`:

~~~js
const Clipboard = require('clipboard')
~~~